service/gcsutils/libtar2vhd/tar2vhd.go

```diff
@@ -80,7 +80,7 @@ func VHD2Tar(in io.Reader, out io.Writer, options *Options) (int64, error) {
 		return 0, err
 	}
 
-	mntFolder, err := ioutil.TempDir("", "mnt")
```

Comment:
the 1st parameter is directory and the 2nd parameter is a prefix  to the 1st parameter, which makes the final string no different. Also, I didn't see anywhere that /tmp set as default.